examples/edi-to-bal/edi_to_bal.md

```diff
@@ -0,0 +1,33 @@
+# EDI to record conversion
+
+EDI is a widely used message format for business-to-business (B2B) communications. Ballerina simplifies working with EDI data by converting them to Ballerina records, so that all operations related to Ballerina records can be applied on EDI data as well (e.g. transforming EDI data, writing EDI data to databases, transering EDI data over various network protocols, etc.).
```

Comment:
```suggestion
EDI is a widely used message format for business-to-business (B2B) communications. Ballerina simplifies working with EDI data by converting them to Ballerina records, so that all operations related to Ballerina records can be applied to EDI data as well (e.g., transforming EDI data, writing EDI data to databases, transferring EDI data over various network protocols, etc.).
```